src/App.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
-
 import './App.scss';
-
+import { PostList } from './components/PostList';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+export const postList = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  comment: comments.filter(comment => comment.postId === post.id),
```

Comment:
метод фильтер возвращает массив
массив комментариев не может называться `comment`
```suggestion
  comments: comments.filter(comment => comment.postId === post.id),
```